std/hash/_wasm/build.ts

```diff
@@ -0,0 +1,60 @@
+// Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
+
+import { encode as base64Encode } from "../../encoding/base64.ts";
+import Terser from "https://cdn.pika.dev/terser@^4.7.0";
```

Comment:
What does terser do?

Reply:
I see - it's a minifier. Is it possible to remove this dependency? Presumably minification doesn't help much?

In std we strive to not have any external dependencies: https://deno.land/manual/contributing/style_guide#minimize-dependencies-do-not-make-circular-imports
In this case, build.ts isn't part of std, it's just a script for generating the wasm, but I think it would be prudent for us to not rely on external code if possible.

Reply:
Also I'm getting an error when running this:
```
> deno run --allow-read --allow-write --allow-run ./build.ts
Download https://cdn.pika.dev/terser@^4.7.0
Download https://cdn.pika.dev/-/terser@v4.7.0-SrAIEQDlaP53b59FLt18/dist=es2019,mode=types/index.d.ts
Download https://cdn.pika.dev/-/terser@v4.7.0-SrAIEQDlaP53b59FLt18/dist=es2019/terser.js
Download https://cdn.pika.dev/-/terser@v4.7.0-SrAIEQDlaP53b59FLt18/dist=es2019,mode=types/tools/terser.d.ts
Download https://cdn.pika.dev/-/source-map@v0.6.1-pyQIuRA3Pg4fA1bvvKWu/dist=es2019/source-map.js
Download https://cdn.pika.dev/-/terser@v4.7.0-SrAIEQDlaP53b59FLt18/dist=es2019,mode=types/tools/terser.d.ts
Download https://cdn.pika.dev/-/source-map@v0.6.1-pyQIuRA3Pg4fA1bvvKWu/dist=es2019,mode=types/source-map.d.ts
Download https://cdn.pika.dev/-/source-map@v0.6.1-pyQIuRA3Pg4fA1bvvKWu/dist=es2019,mode=types/index.d.ts
Compile file:///Users/rld/src/deno/std/hash/_wasm/build.ts
error: Uncaught SyntaxError: Invalid or unexpected token
    at https://cdn.pika.dev/-/terser@v4.7.0-SrAIEQDlaP53b59FLt18/dist=es2019/terser.js:10:342617
```

Reply:
It should make a huge difference if we are targeting wasm. Mostly it would be shortening variable names, the rest simply shouldn't matter in Web Assembly.

Adding minification support into Deno longer term is an objective, but we would want that to come from Rust.

Reply:
I will remove minifier :)